resources/tests/genCerts.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
```

Comment:
Would you tell me how stable it is compared to using our current hard-coded ssl files? Our CI tests (.github/workflows/ci.yaml) rely on the `ubuntu-latest` image. The version of commands (e.g. openssl) on the image will not be changeless in the future. Is there a risk that this file will break?

Reply:
My train of thought on this:

Hard coded certs are black boxes with no hint as to how they were generated. And if they need to change that makes it harder for somebody to create new certs by hand. (for example for my own system I found out that for Vertica 10.x I needed to add `digitalSignature` to keyUsage for the certs to work. That was not required with vertica 9.x. It was trivial for me to add that to my script, as opposed to chasing the right combination of commands to create certs.

I have to think that the `openssl` API is pretty stable, and any change to the existing options would meet with resistance considering how many products document the sequence of commands used in this script.

Reply:
@mjskier Make sense. Thanks for the explanation!